proxyd/server.go

```diff
@@ -586,6 +588,14 @@ func (s *Server) populateContext(w http.ResponseWriter, r *http.Request) context
 	)
 }
 
+func randStr(l int) string {
```

Comment:
Is this used anywhere?

Reply:
Yeah, `populateContext` uses this to generate a `req_id`. It used to live in the file `proxyd/backend_rate_limiter.go` what this PR deleted, so it have to be moved somewhere. I picked `server.go` since its where it is currently used.